receipts/handler/handler.go

```diff
@@ -95,15 +93,12 @@ func (r *ReceiptHandler) GetPendingTxHashList() [][]byte {
 	return hashListCopy
 }
 
-func (r *ReceiptHandler) Close() error {
-	if err := r.leveldbReceipts.Close(); err != nil {
-		return errors.Wrap(err, "closing receipt leveldb")
-	}
-	return nil
+func (r *ReceiptHandler) Close() {
```

Comment:
This function doesn't serve any useful purpose anymore, remove it.

Reply:
ฑำทนอำก